MeshTools.wl

```diff
@@ -150,15 +150,63 @@ MeshElementMeasure[mesh_ElementMesh]:=Module[{
 (*BoundaryElementMeasure*)
 
 
+Clear[elementMeasure]
+
+(* Boundary mesh measure for each submesh. *)
+boundaryElementMeasure[
+	nodes_List/;(Depth[nodes]==4),
+	type_,
+	order_,
+	integrationOrder_]:=
+	Total[boundaryElementMeasure[#,type,order,integrationOrder]&/@nodes]
```

Comment:
Use of Total[...] breaks the intended functionality. We would like to return "BoundaryElement" measure element by element.